### DISCUSSION TITLE
[SOLVED] SyntaxError: Unexpected token '||=' using husky pre-commit

### DISCUSSION THREAD

**Discussion by BJSS-russell-pollock** [vitest-dev/vitest#3515] [Q&A] [answered]
We're switching our tests from jest to vitest.

One person, on the team, cannot use `git commit` because of this error:
```
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Error ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
SyntaxError: Unexpected token '||='
 ❯ wrapSafe internal/modules/cjs/loader.js:1001:16
 ❯ Module._compile internal/modules/cjs/loader.js:1049:27
 ❯ Object.Module._extensions..js internal/modules/cjs/loader.js:1114:10
 ❯ Module.load internal/modules/cjs/loader.js:950:32
 ❯ Function.Module._load internal/modules/cjs/loader.js:790:12
 ❯ Module.require internal/modules/cjs/loader.js:974:19
 ❯ require internal/modules/cjs/helpers.js:101:18
 ❯ Object.<anonymous> node_modules/jsdom/lib/jsdom/living/generated/HTMLInputElement.js:1875:14
 ❯ Module._compile internal/modules/cjs/loader.js:1085:14
 ❯ Object.Module._extensions..js internal/modules/cjs/loader.js:1114:10
```

They have done a fresh clone and fresh install. If they switch back to main, commit works again, it's only the vitest branch. We've checked node/npm versions and whatever else we could think of. Anyone have an idea we can try?

`npm run test` works and so does `npm run lint` (the pre-commit scripts basically)

**Comment by sheremet-va** [comment]
Use higher Node.js version

**Reply by BJSS-russell-pollock** [reply]
Thanks for the suggestion. They tried that and got back to me saying they are getting the same error. We've tried it on five machines and only one has this error which is really hard to pinpoint what's wrong with it.

**Reply by sheremet-va** [reply]
Check your node version when running husky. This error happens because of Node.js executable's version is too low. It's possible that husky uses version different from your terminal.

**Comment by BJSS-russell-pollock** [comment, marked as answer]
@sheremet-va - we sorted it with these basic commands:

```
npm cache clean -force
rm -rf node_modules
reinstall node
```

We're still not sure how or why - but something must've been stuck. Thanks for the help.
